packages/gatsby/src/bootstrap/index.ts

```diff
@@ -60,6 +61,8 @@ export async function bootstrap(
 
   await rebuildSchemaWithSitePage(context)
 
+  saveStateForWorkers([`components`, `staticQueryComponents`])
```

Comment:
I think this should be after `extractQueries` to actually have extracted queries to share? also I do think we should wrap code specific to PQR with some env var already as it doesn't need to execute in regular builds and would be waste of some CPU cycles. Adding flag doesn't make sense yet as feature is not complete and "doesn't do anything" yet, but for internal use and tests env var would work fine I think? something like `process.env.GATSBY_EXPERIMENTAL_PARALLEL_QUERY_RUNNING`?